glean_parser/metrics.py

```diff
@@ -208,9 +214,20 @@ class Timespan(TimeBase):
 @dataclass
 class TimingDistribution(TimeBase):
     typename = 'timing_distribution'
-    # The following is a Gecko-specific property for using the Glean SDK
-    # with GeckoView metrics. See bug 1566356 for more context.
-    gecko_datapoint: str = ''
+
+
+class HistogramType(enum.Enum):
+    linear = 0
+    exponential = 1
+
+
+@dataclass
+class CustomDistribution(Metric):
+    typename = 'custom_distribution'
+
+    range: Tuple[int, int] = field(default_factory=lambda: [0, 60000])
```

Comment:
Any specific reason why we're picking these defaults here, in the parser?

Reply:
Oh -- since these were the defaults picked for the original timing distribution implementation, I thought they were somehow the defaults on desktop.  But it looks from [here](https://searchfox.org/mozilla-central/source/toolkit/components/telemetry/build_scripts/mozparsers/parse_histograms.py) like the only default is that `rangeMin` is 1.

Reply:
Should we have the defaults on the types in a-c ? Also, yes, let's keep the same defaults. Documentation for the desktop part [lives here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/histograms.html#low)

Reply:
Yeah -- I've updated this to have *no defaults*.  I guess to be fully consistent with desktop, though, we would allow `rangeMin` to have a default of 1.

We seem to have a mix of providing defaults in a-c and not.  I think it's best to not have them in a-c, since they wouldn't matter in practice anyway -- glean_parser always generates a value for every parameter when it outputs Kotlin code.  The Kotlin-side defaults would only affect writing Glean's own unit tests.